### ISSUE TITLE
invisible (Diogenes's lamp went out?)

### ISSUE THREAD

**Issue by JDErnest** [pjheslin/diogenes#86]
### Searching for an honest window
It's been a while since I used Diogenes--use to work fine. Now when I open it (on my Dell desktop running Windows 10), it seems to open in an invisible window. Alt-tab thinks it's running, and the taskbar thinks it's running--even shows me a thumbnail of the screen. But I can't get it to display. The usual tricks for getting something back onto your monitor have not worked (using Windows-arrow keys, shift-right-clicking on the taskbar button and then "move"). Rebooting didn't help. Reinstalling didn't help. Help?

**Comment by pjheslin** [comment]
Sorry for the slow response.  I think this is the same as a problem I have encountered on my Macbook.  When I use an external monitor and have Diogenes open on that monitor and then close Diogenes, and then later restart Diogenes on the laptop without the monitor, Diogenes remembers where it was last open and opens there, but it is now in an inaccessible location, because the external monitor is no longer available.

If that is your problem, I have added a fix in the forthcoming version: a menu item that allows you to open a new window not in the previous position, but at the top left corner of the current screen.  

Until that version comes out, an interim fix would be to delete the file windowstate.json, where Diogenes remembers where on the screen it was last opened.

On a Mac this is located at: /Users/USERNAMELibrary/Application\ Support/Diogenes/windowstate.json

On Windows I think it would be found at: C:\Users\USERNAME\AppData\Roaming but I am not perfectly sure.  If you type  %AppData% into the address bar, that should point you to the right place.
